README.md

```diff
@@ -1,10 +1,33 @@
+# NimbleTemplate
+
+[![Test template](https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml/badge.svg?branch=develop)](https://github.com/nimblehq/elixir-templates/actions/workflows/test_template.yml)
+
+
+> A Library for initialize new Phoenix projects
```

Comment:
Since we merged the Mix variant, we also support non-Phoenix projects too @byhbt